RecoTracker/TkSeedGenerator/python/GlobalSeedsFromPairsWithVertices_cff.py

```diff
@@ -12,7 +12,7 @@
 from RecoTracker.TkSeedingLayers.PixelLayerPairs_cfi import *
 from RecoTracker.TkSeedingLayers.MixedLayerPairs_cfi import *
 
-from RecoTracker.TkTrackingRegions.GlobalTrackingRegionWithVertices_cfi import *
+from RecoTracker.TkTrackingRegions.globalTrackingRegionWithVertices_cfi import *
```

Comment:
In principle the proper file to import from is `globalTrackingRegionWithVertices_cff` which has bunch of era customizations on top of the auto-generated one
https://github.com/cms-sw/cmssw/blob/29ff9cba70150b1b66144603a47e376fdfac2183/RecoTracker/TkTrackingRegions/python/globalTrackingRegionWithVertices_cff.py#L3-L20

Maybe we should take a look on how to auto-generate `globalTrackingRegionWithVerticesDefault_cfi` (or similar) to have a file with customizations called `globalTrackingRegionWithVertices_cfi`.